src/app/Dashboard/Dashboard.tsx

```diff
@@ -270,26 +310,33 @@ export const Dashboard: React.FC<DashboardProps> = (_) => {
 
   return (
     <TargetView pageTitle="Dashboard" compactSelect={false}>
-      <Grid id={'dashboard-grid'} hasGutter>
-        {cardConfigs.map((cfg, idx) => (
-          <GridItem span={cfg.span} key={cfg.id} order={{ default: idx.toString() }}>
-            {React.createElement(getConfigByName(cfg.name).component, {
-              ...cfg.props,
-              dashboardId: idx,
-              actions: [
-                <DashboardCardActionMenu
-                  key={`${cfg.name}-actions`}
-                  onRemove={() => handleRemove(idx)}
-                  onResetSize={() => handleResetSize(idx)}
-                />,
-              ],
-            })}
+      <ChartContext.Provider value={{ controller: chartController.current }}>
```

Comment:
Context value here should be wrapped in `React.useMemo`. Right now, every time the card is re-order or resize, context value is always a different object -> lots of observable is being re-subscribed.

I think this might be the issue for https://github.com/cryostatio/cryostat-web/pull/856#discussion_r1100888788.